libs/perun/components/src/lib/groups-list/groups-list.component.html

```diff
@@ -113,12 +111,13 @@
         </th>
         <td *matCellDef="let group" mat-cell>
           <i
-            class="material-icons {{getStatusAttribute(group) | groupStatusIconColor}}"
-            matTooltip="{{getStatusAttribute(group)}}"
+            *ngIf="{status: group.attributes | findAttribute: 'groupStatus'} as groupStatus"
+            class="material-icons {{groupStatus.status | memberStatusIconColor}}"
```

Comment:
I think there should be `groupStatusIconColor` instead of `memberStatusIconColor` or the `memberStatusIconColor` pipe has to be changed, because now it requires RichMember as an argument, not a string.

Reply:
I have used the groupStatusIconColor as is it on master.